### DISCUSSION TITLE
Can't remove a repository

### DISCUSSION THREAD

**Discussion by EVALORE** [community/community#128767] [Repositories]
### Select Topic Area

Bug

### Body

for some reason i have a repository which i cant remove from my list of repositories, because its counted but not shown in the list of the repositories on github. however, if i use github desktop or api to list repositories i cant see it. Is there a way to remove it completely?

**Comment by davevad93** [comment]
Hi @EVALORE ,

Try waiting a few hours, sometimes when a repo is deleted it's still counted in the repos list even though it's not really there anymore. Some data on GitHub takes a while to update, don't worry, it's normal.

**Reply by EVALORE** [reply]
the problem is that repo is still listed for more than one year, and still doesn't disappear

**Reply by davevad93** [reply]
Well, that's definitely not normal, try with the following and see if that fixes it:

1. Go to `Settings`.
2. Scroll down in the sidebar and click on `Repositories`.
3.  Click on `Deleted repositories` and restore the deleted repo, then delete it again and wait a while.
![0](https://github.com/community/community/assets/94308234/4ad10b42-57bc-46e1-9835-88e65e445a96)

If by doing that the issue persists, contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

**Reply by davevad93** [reply]
Sorry, you can't do that, I forgot you said that repository was deleted 1 year ago and you can only restore repos within 90 days. So contact support directly hoping they can fix it for you.
